File: Exercicios/Estruturas Condicionais/1051.py
valor = float(input())
if valor <= 2000:
    print('Isento')
elif valor <= 3000:
    imposto = (valor - 2000) * 8/100 
    print(f'R$ {imposto:.2f}')
elif valor <= 4500:
    imposto = (valor - 3000) * 18/100 + 80 #1000 * 8/100
    print(f'R$ {imposto:.2f}')
else:
    imposto = (valor - 4500) * 28/100 + 270 + 80 # 1500 * 18/100, 1000 * 8/100
    print(f'R$ {imposto:.2f}')

File: 2025/intermediario/2025.1/1051.cpp
#include <bits/stdc++.h>

using namespace std;

int main(){
    double valor, imposto;
    cin >> valor;
    if (valor <= 2000){
        cout << "Isento" << "\n";
        return 0;
    } else if (valor <= 3000){
        imposto = (valor - 2000) * 0.08;
    } else if (valor <= 4500){
        imposto = (valor - 3000) * 0.18 + 80;
    } else {
        imposto = (valor - 4500) * 0.28 + 80 + 270;
    }
    cout << fixed << setprecision(2) << "R$ " << imposto << "\n";
    return 0;
}